### DISCUSSION TITLE
Training Lora based on PixArt-XL-2-1024-MS

### DISCUSSION THREAD

**Discussion by Sainzerjj** [PixArt-alpha/PixArt-alpha#95] [Q&A]
I am training Lora on PixArt-XL-2-1024-MS using the code from this repository, using text prompts like "1girl, solo, short hair, black hair, hair ornament, dress, bare shoulders, brown eyes, jewelry," and found that although I set the learning rate very large, it still need to train 1000+ steps, I want to ask if this is normal or is there an additional training technique?

**Comment by lawrence-cj** [comment]
This may be caused by the small graident clip.
